server/prisma/generator/factories/instanceRoles.factory.ts

```diff
@@ -15,7 +15,13 @@ const roles: InstanceRole[] = [
     // the owners should be able to do everything
     permissions: allPermissions,
   },
-  { name: 'member', permissions: [] },
+  {
+    name: 'member',
+    permissions: [
+      InstancePermission.ChapterJoin,
+      InstancePermission.ChapterSubscriptionsManage,
+    ],
```

Comment:
Giving members permissions does create some potential awkwardness around bans.  I think it's fine for now, since we only have chapter level bans at the moment, but I just thought I'd mention it. We do need to be very careful with these.